syntheseus/cli/eval_single_step.py

```diff
@@ -397,7 +405,7 @@ def print_and_save(results: EvalResults, config: EvalConfig, suffix: str = "") -
         if f.name not in ("model_info", "top_k", "predictions", "back_translation_predictions"):
             print(f"{f.name}: {getattr(results, f.name)}")
 
-    print(f"top_k results {suffix}:")
+    print(" ".join(["top_k results", suffix]) + ":")
```

Comment:
Eh, my intention was that if `suffix` is empty then it should be `"top_k results:"` instead of `"top_k results :"` (no extra space) but that didn't work out... I can just have an `if` then

Reply:
Up to you: this is just a nitpick. Feel free to change or just mark as resolved and merge.

Reply:
The entirety of my changes to this line was just trying to get rid of the extra space (by making the code slightly more complex), so I'm not going to leave it like this 😄 Fixed now.